File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
import java.util.*;

class Solution {
    public int solution(int price) {
        int answer = 0;
        if(price >= 500000) {
            answer = (int) (price * 80 / 100);
        }
        else if (price >= 300000) {
            answer = (int) (price * 90 / 100);
        }
        else if (price >= 100000) {
            answer = (int) (price * 95 / 100);   
        }
        else {
            answer = price;

        }
        return answer;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
import math
def solution(price):
    if price<100000:
        answer=price
    elif 100000<=price<300000:
        answer=price*0.95
    elif 300000<=price<500000:
        answer=price*0.90
    elif price>=500000:
        answer=price*0.80
    return int(answer)